### PR TITLE
UPS image update

### PR DESCRIPTION
**Pull request by matzew**
New image, fixes some 'awful font' issues - basically getting UPS release bits from this week

### CODE REVIEW

File: roles/provision-unifiedpush-apb/defaults/main.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ups_secret_name: "unified-push-server"
-ups_image_version: "1.2.1.3"
\ No newline at end of file
+ups_image_version: "1.2.2"
```
Comment (philbrookes) [inline roles/provision-unifiedpush-apb/defaults/main.yml]: it looks like these vars aren't actually used?
https://github.com/matzew/unifiedpush-apb/blob/9ff7ddd8555388c8d845bbb47780f1109a7451af/roles/provision-unifiedpush-apb/tasks/main.yml#L104

Comment (matzew) [reply roles/provision-unifiedpush-apb/defaults/main.yml]: rhmmmmmmmmmmmmm

**Comment by david-martin** [comment]
@matzew I can give this a spin later

**Comment by matzew** [comment]
@david-martin thanks

for some reason it's working for me (LOL), but not for @philbrookes :confused:

**Comment by matzew** [comment]
merging.  For @david-martin it was booting correctly -> 

https://gist.github.com/david-martin/39b42b96f00b8d7d20b2c4013cd8a457

**Pull request merged**